store/slo.go

```diff
@@ -27,7 +27,7 @@ func (cs *SLOStore) createTableIfNotExists() {
 	fmt.Println("Creating the slo table now!")
 	if !cs.DB.Migrator().HasTable(&schema.SLO{}) {
 		if err := cs.DB.Migrator().CreateTable(&schema.SLO{}).Error; err != nil {
-			fmt.Println(err)
+			fmt.Println(err())
```

Comment:
this looks very weird and not like normal code. Any special reasons for doing this? Also I am not sure how this doesn't trigger a panic if no error occurs.

Normally I would expect code like this:

```
if err := cs.DB.Migrator().CreateTable(&schema.SLO{}); err != nil {
			fmt.Println(err)
```

`fmt` knows about the error type and should automatically invoke `Error()` to get a string.

Reply:
I felt the same way, but this was how I made `go test` errors go away. Happy to remove them if there's a better solution to that.

Reply:
```
$ make test
docker run --rm -v /home/jseekins/github/slo-tracker:/app:z -w /app golang:1.17 go test -race -mod=vendor ./...
# slo-tracker/store
store/slo.go:30:4: Println arg err is a func value, not called
```

Reply:
@johnseekins not necessarily a problem /critique with your PR but with the `master` branch. Just wanted to point it out.

Reply:
@johnseekins Thanks for your contribution :)
As @zeisss pointed out, Can you please revert back `err()` to `err`.
Everything else is looking good.